index_hide_quest_answ.js

```diff
@@ -0,0 +1,144 @@
+
+
+
+
+//resolve // QUESTION 1 here
+
+var newDiv = document.createElement('div');
+var a1Div = document.getElementById('a-1');
+a1Div.append(newDiv);
+
+newDiv.className = 'square';
+newDiv.innerText = 'x';
+
+newDiv.addEventListener('click', () =>  {
+    if (newDiv.innerText === 'x') {
+        newDiv.innerText = 'o';
+    } else {
+        newDiv.innerText = 'x';
+    }
+});
+
+
+//------------------------Question 2 ---------------------------
+//Write a JavaScript program to remove items from a dropdown list.
+//In order to create the list we are providing you with the array bellow, those items are the ones you need to add to the dropdown list as the select options
+//you also need to create a button that makes the remove action of the element selected
+// append the new list as a child of the element with the 'id = a-2'
+// append the button as a child of the element with the 'id = a-2'
+
+// declare the variables
+const colors = ['red' , 'white', 'black' , 'green' , 'orange'];
+const select = document.createElement('select');
+const a2Div = document.getElementById('a-2');
+
+// using for loop create dropdown list
+for (i = 0; i < colors.length; i++) {
+    let optionValue = document.createElement('option');
+    optionValue.value = colors[i];
+    optionValue.innerText = colors[i];
+    select.appendChild(optionValue);
+}
+
+// append the new list as a child of the element with the 'id = a-2'
+a2Div.appendChild(select);
+
+// create Remove button
+const removeButton = document.createElement('button');
+removeButton.innerText = 'Remove';
+
+// removing elements bi the index in the colors list
+removeButton.addEventListener('click', () => {
+    select.options[select.selectedIndex].remove();
+})
+
+// append the button as a child of the element with the 'id = a-2'
+a2Div.appendChild(removeButton);
+
+
+//------------------------Question 3 ---------------------------
+//Now lets mix some maths and programming. In the function below (calculate_sphere) write a JavaScript program to calculate the volume and surface area of a sphere.
+//take the radius from the input value element with the 'id=radius' in the index.html file
+//And display the results in the elements with the 'id = volume' and 'id = area' respectively
+//hint: the volumen of a sphere is ((4/3) × π × r^3) and the surface area is (4 × π × r^2)
+
+// declare the variables that is going to use in function
+const radiusField = document.getElementById('radius');
+const volumeField = document.getElementById('volume');
+const areaField = document.getElementById('area');
+
+const calculate_sphere = () => {
+
+    // input the radius value 
+    let radius = radiusField.value;
+
+    // calculate the volume and area value
+    let volume = ((4/3) * Math.PI * (radius^3));
+    let area = (4 * Math.pow(radius.value, 2));
```

Comment:
In line 76 "radius = radiusField.value" already assigned to radius but in line 80 it called again as "radius.value" second time it must be called as "radius".  In line 80 "Math.PI" is missing to make formula calculates the area.In line 79 Math.pow() can be used to calculate exponent.

Reply:
@sinancengiz, thank you so much for your comments. Of course, I didn't check the code within question 3 after rewriting one more time and got an error as a result. I made changes into index_hide-quest_answ.js and index.html. I believe, now it should work